### PR TITLE
STAC-3401 blacklisted processes

### PR DESCRIPTION
**Pull request by zandrewitte**
### What does this PR do?

A brief description of the change being made with this pull request.
Adds default process blacklist of process that are typically uninteresting. 

### Motivation

What inspired you to submit this pull request?
STAC-3401 Story to filter uninteresting processes
### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -621,7 +621,49 @@ process_config:
...
-#   blacklist_patterns:
+  blacklist_patterns:
+  # Linux Specific patterns
+    - "^/sbin/"
+    - "^/usr/sbin/"
+    - "^/usr/bin/vi(?:m|m.basic)?$"
+    - "^sshd:"
+    - "^/bin/bash"
+    - "^/usr/bin/bash"
+    - "^/usr/lib/systemd/*"
```
Comment (TizianoPerrucci) [inline pkg/config/config_template.yaml]: is a `*` needed ?
I see other example like `^/usr/sbin` which correctly ignores sshd (as I can see from the molecule expectations)

File: pkg/config/config_template.yaml [hunk 2]
```diff
@@ -621,7 +621,49 @@ process_config:
...
+    - "^/lib/systemd/*"
+    - "^/usr/bin/dbus-daemon"
+    - "^/usr/bin/tail"
+    - "^-bash"
+    - "^\\(sd-pam\\)"
+    - "^pickup"
+    - "^qmgr"
+  # Windows Specific patterns
+    - "cmd.exe"
```
Comment (TizianoPerrucci) [inline pkg/config/config_template.yaml]: can we order them alphabetically please :) ?

File: pkg/config/config_template.yaml [hunk 3]
```diff
@@ -621,7 +621,49 @@ process_config:
     process: 30
     connections: 30
 #   A list of regex patterns that will exclude a process if matched.
-#   blacklist_patterns:
+  blacklist_patterns:
+  # Linux Specific patterns
+    - "^/sbin/"
```
Comment (TizianoPerrucci) [inline pkg/config/config_template.yaml]: can we order them alphabetically please :) ?

File: pkg/config/config_template.yaml (hunk 1 above)
Comment (zandrewitte) [reply pkg/config/config_template.yaml]: Mmm yes I think the * is not needed. I'll double check. The comparison / matching happens on the first argument of the cmd line. So in the case of `sshd` it actually matches only for `^sshd:`

**Pull request merged**